fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -918,6 +915,30 @@ public void pageOrientation(String orientation) {
         this.pageOrientation = orientation;
     }
 
+    public void repeatRows(int startRow, int endRow) {
+        this.repeatingRows = new ArrayList<Integer>();
+        this.repeatingRows.add(startRow);
+        this.repeatingRows.add(endRow);
+    }
+
+    public void repeatRows(int row) {
+        this.repeatingRows = new ArrayList<Integer>();
+        this.repeatingRows.add(row);
+        this.repeatingRows.add(row);
+    }
+
+    public void repeatCols(String startCol, String endCol) {
+        this.repeatingCols = new ArrayList<String>();
```

Comment:
fastexcel's API uses integer column indexes everywhere else. 
Use `Range.colToString(int)` to convert column index to column name where writing to xml.

Reply:
Makes sense, will fix that.